src/cmd/tools/split_shards/README.md

```diff
@@ -0,0 +1,23 @@
+# split_shards
+
+`split_shards` is a command line utility that traverses M3DB data folder
+and splits data filesets into more granular shards.
+
+# Usage
+
+```
+$ git clone git@github.com:m3db/m3.git
+$ make split_shards
+$ ./bin/split_shards
+Usage: split_shards [-b value] [-d value] [-f value] [-h value] [-s value] [parameters ...]
+ -b, --block-until=value
+       Block Until Time [in nsec]
+ -d, --dst-path=value
+       Destination path prefix [e.g. /var/lib/m3db/data]
+ -f, --factor=value
+       Integer factor to increase the number of shards by
+ -h, --src-shards=value
+       Original (source) number of shards
+ -s, --src-path=value
+       Source path [e.g. /temp/lib/m3db/data]
```

Comment:
Should this have the trailing `/data` at the end? Usually for the "PathPrefix" that's given to tools we just set the root path prefix path as command line arg (i.e. /var/lib/m3db not /var/lib/m3db/data) and let the tool find the subdirectories it needs.

Reply:
My idea was that it should, because I traverse all the subdirectories.
This allows to process all the namespaces in one go (`/var/lib/m3db/data`), and also supports processing of a specific namespace (if we want to do it one at a time for some reason), by specifying `/var/lib/m3db/data/aggregated_30d` for example.
And then, if I start traversing from `/var/lib/m3db`, this also includes reverse index filesets which I would not want to mess with.